### ISSUE TITLE
Temporary charge description in scene summary

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#179]
At the moment, the scene summary charge description is blank.  Design has yet to be reviewed, but something will be needed.  A simple placeholder should be fine for now.

**Comment by jessegreenberg** [comment]
Done.
